includes/Modules/Site_Verification.php

```diff
@@ -96,6 +101,12 @@ function () {
 			}
 		);
 
+		add_action(
+			'googlesitekit_invalidate_verification_meta_cache',
```

Comment:
This should instead hook into the core actions `update_user_meta` and `add_user_meta`. It requires minimally more code (we need to check `meta_key`), but it allows us to decouple this caching mechanism completely from the plain verification meta storage itself.